molecule/default/verify.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Check aau data file exists
+  hosts: all
+  become: yes
+
+  tasks:
+    - name: Check aau data file exists
+        stat:
```

Comment:
This is an ineffective test by itself. Stat doesn't fail if the path doesn't exist. Whether the path exits is a property of the data returned by the stat module. 

If you continue beyond this and create a an assertion task, then it can work:

```
    - assert:
        that:
          - stat_result.stat.exists
```